src/librustc_session/options.rs

```diff
@@ -283,12 +284,14 @@ macro_rules! options {
             Some("one of: `disabled`, `trampolines`, or `aliases`");
         pub const parse_symbol_mangling_version: Option<&str> =
             Some("either `legacy` or `v0` (RFC 2603)");
+        pub const parse_src_file_checksum: Option<&str> =
+            Some("either `none`, `md5`, or `sha1`");
```

Comment:
What are the chances we can avoid supporting MD5 altogether? What *needs* it?

Reply:
Nevermind, already answered in https://github.com/rust-lang/rust/pull/69718#discussion_r388604692.